test/models/lookup_value_test.rb

```diff
@@ -35,23 +41,27 @@ def valid_attrs2
     end
   end
 
-  test "non-admin user can view only his hosts allowed by filters" do
+  test "non-admin user cannot create lookup value for hosts he does not have allowed by filters" do
+    skip('lookup value does not currently check permissions')
     # Host.authorized(:view_hosts, Host) returns only hosts(:one)
     user = users(:one)
     role = FactoryBot.create(:role, :name => 'user_view_host_by_ip')
     FactoryBot.create(:filter, :role => role, :permissions => [Permission.find_by_name(:view_hosts)], :search => "name = #{@host1.name}")
     # Todo, restore the ip test variant once our scoped-search works with host.ip again
     # FactoryBot.create(:filter, :role => role, :permissions => [Permission.find_by_name(:view_hosts)], :search => "ip = #{@host1.ip}")
     user.roles << [role]
+    attrs = valid_attrs1 # create key outside as_user
     as_user :one do
       assert Host.authorized(:view_hosts, Host).where(:name => @host1.name).exists?
       refute Host.authorized(:view_hosts, Host).where(:name => @host2.name).exists?
+      lookup_value = LookupValue.new(attrs)
+      refute lookup_value.save
```

Comment:
Looks like this test changed, the original test didn't have anything to do with LVs, but the new test doesn't actually work and is skipped - should we just remove the test completely?

Reply:
Well I've fixed the test to test what I believe it was supposed to test, but the tested functionality is not there.
I'm not sure though if we shouldn't check permissions to at least see the related records, while creating the `LookupValue`.
I think it would make sense, so I've kept the test.

Reply:
still keeping the test. Do you think we should get rid of it?

Reply:
considering it isn't implemented in the model right now, i'd say drop. 
tbh i'm not sure this check even makes sense, a user could have e.g. `edit_external_parameters` permission that allows them to create matchers for fqdns they can't see

Reply:
Yeah, it's around for quite some while and noone complained yet.
It could have some security risks though. Should we create at least a redmine issue?